docs/source/docker/getting_started/setup.rst

```diff
@@ -168,8 +168,10 @@ You should see an output similar to this one:
     [2022-05-02 20:37:27] Congratulations! It looks like the Lightly container is running!
 
 
-Start the Lightly Worker
-^^^^^^^^^^^^^^^^^^^^^^^^
+.. _ref-worker-register:
+
+Register the Lightly Worker
```

Comment:
Start is the more accurate word here. Registering is what you do when you get the `worker_id`